tpp-sample/src/EidasMethodsSampleTest.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Collections.Generic;
+using Org.BouncyCastle.Asn1;
+using Org.BouncyCastle.Asn1.X509;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Operators;
+using Org.BouncyCastle.Math;
+using Org.BouncyCastle.Security;
+using Org.BouncyCastle.X509;
+using Tokenio.Proto.Common.AliasProtos;
+using Tokenio.Security.Utils;
+using Xunit;
+
+namespace Tokenio.Sample.Tpp
+{
+    public class EidasMethodsSampleTest
+    {
+        [Fact]
+        public void VerifyEidasTest()
+        {
+            using (Tokenio.Tpp.TokenClient tokenClient = TestUtil.CreateClient())
+            {
+                var tppAuthNumber = RandomNumeric(15);
+                var keyPair = GenerateKeyPair();
+                string certificate = GenerateCert(keyPair, tppAuthNumber);
+                Member verifiedTppMember = EidasMethodsSample.VerifyEidas(
+                    tokenClient,
+                    tppAuthNumber,
+                    certificate,
+                    "wood",
```

Comment:
Make it "gold" please ("wood" will cause an error since today)

Reply:
done